src/main/java/com/ita/if103java/ims/dao/impl/ItemDaoImpl.java

```diff
@@ -62,6 +62,17 @@ public Item findItemByAccountId(Long id) {
 
     }
 
+    @Override
+    public Item findItemById(Long id) {
+        try {
+            return jdbcTemplate.queryForObject(Queries.SQL_SELECT_ITEM_BY_ID, itemRowMapper);
+        } catch (EmptyResultDataAccessException e) {
+            throw itemEntityNotFoundException(e.getMessage(), "id = " + id);
+        } catch (DataAccessException e) {
+            throw crudException(e.toString(), "get", "id  = " + id);
```

Comment:
We agreed not to call methods here, but directly throw new exception.

Please change it here and in all other places